SemApps News #1

srosset81 opened the pull request:
Closes #374 

Le travail est prévisualisable ici:
https://website-git-semapps-news-1.semapps.vercel.app/blog

L'idée serait de publier la newsletter sur le site (blog) et aussi par email aux personnes intéressées.
On pourrait la traduire en anglais pour le site, et l'envoyer en français par email.
Je propose qu'on travaille toujours par PR, afin de faciliter les contributions.

Pour l'instant j'ai juste fait le squelette et j'ai rempli quelques cases où ma contribution était la plus pertinente.

@ReX-AV-Gab je te laisse regarder cette proposition, faire des modifications et m'appeler s'il faut en parler.

@ValentineMathieu pourra nous aider à la finaliser et (peut-être ?) à l'envoyer début octobre.

srosset81 committed:
Add page

srosset81 committed:
Add to menu

ReX-AV-Gab commented:
Je me charge de solliciter Simon et Alice.

srosset81 commented:
Ping @GuillaumeAV @bouviermullerp @simonLouvet
N'hésitez pas à ajouter des éléments, ou à remplir les cases vides si vous voulez aider.
L'idée serait que la newsletter parte en début octobre.

Alice-Pog committed:
Update on the meet up, SIB et Solid

GuillaumeAV committed:
Ajouts liens vidéos + dédicace Seb ;)

GuillaumeAV commented on website/blog/2019-09-25-semapps-news-1.md:
```diff
@@ -0,0 +1,110 @@
+
+Une quinzaine de contributeurs de [YesWiki](https://yeswiki.net) se sont retrouvés près d'Avignon en fin septembre et une des nouveautés qui est sortie de cette rencontre est un système d'API qui permet de facilement ajouter, éditer ou supprimer des données via de simples appels HTTP.
+
+YesWiki était déjà compatible web sémantique, avec la possibilité de sortir les données en JSON-LD. Avec cette nouvelle avancée, YesWiki devient un serveur LDP à part entière, ainsi que le montre cette interface SemApps qui utilise un YesWiki comme base de donnée.  
```
Il manque le lien ;)

srosset81 replied:
Oui, en fait il y a pas encore d'exemples à montrer, alors j'ai plutôt adapté le texte.

srosset81 committed:
Fix link

srosset81 committed:
Add link and adapt YesWiki text

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

ajout des liens vers le power point et la documentation de la présentation SemApps/PAIR à l'UTT

srosset81 commented:
On y est presque ! @ReX-AV-Gab tu serais motivé pour remplir la section "Avancées humaines" ?
@simonLouvet on attend toujours un petit paragraphe sur DFC...

simonLouvet committed:
add DFC text

simonLouvet commented:
@srosset81 done

ValentineMathieu reviewed (commented):
Pour l'intro : Pour éviter la répétition "a pour vocation" peut être remplacer par : "Bienvenue dans cette première newsletter de SemApps, ayant pour vocation de vous tenir au courant de nos avancées, de manière synthétique et accessible. L'équipe SemApps prévoit de l'éditer de manière régulière, tous les 2-3 mois."

srosset81 committed:
Valentine's suggestion

scenaristeur commented:
Hello Les amis, je suggère Fuseki et non Fuseki 😉
+ 
Data food: Semapps a permis de migrer d'un base MongoDB --> on dit unE base

Et Les Données , sans majuscule

J9rem commented on website/blog/2019-09-25-semapps-news-1.md:
```diff
@@ -0,0 +1,115 @@
+
+## Avancées humaines
+
+#### Jérémy Dufraisse nous a rejoint
```
```suggestion
#### Jérémy Dufraisse nous a rejoint

Passionné et débutant en développement de sites internet, Jérémy découvre l'équipe SemApps avec l'intention de s'impliquer davantage, en particuliers sur le développement du code SemApps.
Membre du cercle-cœur des Colibris Lorient, il participe actuellement à la plateforme version 0 des Chemins de la Transition, projet de l'écosystème SemApps.
```
Voici une proposition de texte pour la newsletter pour  la partie me concernant.

srosset81 replied:
Parfait, merci ! :-)

srosset81 committed:
Insert Jeremy's suggestion

Co-authored-by: Jérémy Dufraisse <67152407+J9rem@users.noreply.github.com>

ReX-AV-Gab committed:
Update 2019-09-25-semapps-news-1.md

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

ajout de la version anglaise pour le docusaurus

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

suppression de la version française (copiée sur le PAD que j'avais créé pour la version anglaise)

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

petite modif

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

À la demande d'Alice, j'ajoute un petit point sur la campagne de dons de l'AV.

ValentineMathieu commented:
@assemblee-virtuelle/semapps Hello tout le monde ! comme vous pouvez le voir, j'ai rajouté un point sur la campagne de dons à la demande d'Alice. J'ai traduit et modifié son texte à ma manière, libre à vous d'apporter vos commentaires ou de changer son emplacement :) Petite question, @Alice-Po m'a demandé de mettre un lien pour l'adhésion et un lien pour les dons mensuels, mais je ne trouve pas la page adhésion si quelqu'un peut m'éclairer ? j'ai l'impression qu'il n'y a que les dons sur le site de l'AV. Merci !

srosset81 commented on website/blog/2019-09-25-semapps-news-1.md:
```diff
@@ -114,5 +114,11 @@ Here are two working groups where it would be nice to have representatives from
 
+#### Virtual Assembly's donation campaign 
+
+The Virtual Assembly's organization mainly relies on volunteering. In order to maintain its sustainability, its research and development activities as well as its independance, the VA count on your donations. The more you contribute regularly, the more our association strengthens economically. In this way, we encourage you to choose monthly donations rather than ponctual ones. In any case, we wil ...
```
Top !

simonLouvet commented:
a on une date de publication?

srosset81 commented:
> Petite question, @Alice-Po m'a demandé de mettre un lien pour l'adhésion et un lien pour les dons mensuels, mais je ne trouve pas la page adhésion si quelqu'un peut m'éclairer ? j'ai l'impression qu'il n'y a que les dons sur le site de l'AV. Merci !

Oui je crois que pour le moment c'est comme ça. Nous avons une issue https://github.com/assemblee-virtuelle/vie-asso-av/issues/25 qui a pour but de plus mettre en avant les adhésions payantes. @GuillaumeAV ?

ValentineMathieu commented:
> > Petite question, @Alice-Po m'a demandé de mettre un lien pour l'adhésion et un lien pour les dons mensuels, mais je ne trouve pas la page adhésion si quelqu'un peut m'éclairer ? j'ai l'impression qu'il n'y a que les dons sur le site de l'AV. Merci !
> 
> Oui je crois que pour le moment c'est comme ça. Nous avons une issue [assemblee-virtuelle/vie-asso-av#25](https://github.com/assemblee-virtuelle/vie-asso-av/issues/25) qui a pour but de plus mettre en avant les adhésions payantes. @GuillaumeAV ?

Ok en attendant j'enlève la partie adhésion et je propose le lien ponctuel :)

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

petite modif en attendant la réponse sur les adhésions

ValentineMathieu commented:
> a on une date de publication?

tout dépend de si on souhaite attendre la publication de la NL de l'AV avec un lien vers une inscription à la NL de SemApps, ou pas. Débat ouvert [ici](https://github.com/assemblee-virtuelle/semapps/issues/374)

srosset81 commented on website/blog/2019-09-25-semapps-news-1.md:
```diff
@@ -118,7 +118,7 @@ If you want to share information about the Solid ecosystem, there is a dedicated
-* I would like to take a [membership]()
-* I would like to make a [donation](https://www.virtual-assembly.org/faire-un-don/)
+* I would like to make a [regular donation](https://www.virtual-assembly.org/faire-un-don/)
+* I would like to make a [ponctual donation](https://www.virtual-assembly.org/faire-un-don/)
```
C'est "punctual", avec un "u", mais DeepL suggère plutôt "one-time donation".

ValentineMathieu committed:
Update 2019-09-25-semapps-news-1.md

modif proposée par Sébastien

GuillaumeAV commented:
Helloo tout le monde ! 
Je viens de regarder, à l'époque, l'adhésion supposait de souscrire au code social de l'AV, c'était une adhésion morale, il n'était pas nécessaire de payer pour adhérer, mais il était en revanche possible d'ajouter un don à son adhésion avec l'avantage de pouvoir défiscaliser 66 de son don. 

Quand je suis allé voir l'interface en ligne, il n'y a plus de case ni d'onglet adhérer ... 

Je vois 3 solutions : 
- On communique sur la campagne de dons, et pas sur les adhésions : **Le plus simple à mon avis ...**
- On refait une interface pour adhérer sur hello asso et wordpress ... (Techniquement c'est quelques minutes, mais le design, surtout s'il est collaboratif, pourrait prendre plus de temps...)

srosset81 committed:
Update 2019-09-25-semapps-news-1.md

srosset81 commented:
Je merge sur le site vu que la newsletter a été publiée :-)
